src/loadMessages.js

```diff
@@ -0,0 +1,11 @@
+export default function loadMessages(lang) {
+  const wrapped = import(`./messages/${lang}`)
+
+  if (typeof window !== 'undefined') {
+    return wrapped.then(msgs => msgs.default)
+  }
+
+  /* eslint-disable global-require, import/no-dynamic-require, prefer-template */
+  // return Promise.resolve(require('./messages/' + lang).default)
+  return __webpack_require__(wrapped.resolve()).default
+}
```

Comment:
I figure this:

```js
const chunkNames = flushChunknames()
chunkNames.push(wrapped.chunkName())
const { scripts } = flushChunks(stats, { chunkNames })
```

is not really an option. What I am looking for is a far more general approach where this kind of loading just works.